snuba/processor.py

```diff
@@ -112,7 +114,7 @@ def _floatify(s) -> Optional[float]:
         return None
 
 
-def _unicodify(s) -> Optional[str]:
+def _unicodify(s: Any) -> Optional[str]:
```

Comment:
This seems something that should be stricter than Any

Reply:
What do you suggest? I understand this function as needing able to take any user provided input and turn it into a valid string